src/tools/pontoon/adding_new_locale.md

```diff
@@ -101,4 +101,4 @@ Population: 1863468
 
 Find the locale code in [CLDR territoryInfo.json](https://github.com/unicode-cldr/cldr-core/blob/master/supplemental/territoryInfo.json) and multiply its `_populationPercent` with the territory `_population`. Repeat if multiple occurrences of locale code exist and sum products.
 
-At this point click **SAVE** in the bottom right corner to save and create the new locale. Then, from the [standard admin interface](https://pontoon.mozilla.org/admin/), enable at least one “non system project” (i.e. not `Tutorial` or `Pontoon Intro`) and wait until Pontoon syncs for the locale dashboard to work. You can then check if the locale’s page, as well as the new project, are available - in this case at https://pontoon.mozilla.org/am/
+At this point click **SAVE** in the bottom right corner to save and create the new locale. The "Terminology" project should then appear under the new team's page. Wait until Pontoon syncs for the locale dashboard to work. You can then check if the locale’s page, as well as the new project, are available - in this case at https://pontoon.mozilla.org/am/
```

Comment:
I'm confused by this change. Is this really correct? Terminology is a normal project, it won't appear if you don't add the locale, and I'm not aware of anything that adds locales automatically.

Reply:
I did not need to enable the Terminology project from standard admin interface (https://pontoon.mozilla.org/admin/)  as states the documentation. Terminology project was created on Pontoon once I created "ann" locale from pontoon/a/ - without me adding it
(https://pontoon.mozilla.org/ann/)

Reply:
I realized I could test this on stage, and it's indeed the case. So, the question becomes "when did it change"?
https://mozilla-pontoon-staging.herokuapp.com/dory/

Reply:
Confirmed! Terminology is created for newly added locales:
https://github.com/mozilla/pontoon/blob/f52d415/pontoon/base/signals.py#L170-L182